libs/clients/syslumenn/src/lib/syslumennClient.service.ts

```diff
@@ -425,4 +427,15 @@ export class SyslumennService {
     })
     return res
   }
+
+  async getEstateInfo(nationalId: string): Promise<EstateInfo[]> {
```

Comment:
Just a note: Hopefully we are cautious regarding who can look the estate info up and what can be looked up (a user perhaps shouldn't be able to get data for every national id).

Reply:
This is a good note.
But this isn't exposed, rather accessed through the service only at the application creation time.